root/static/scripts/edit/forms.js

```diff
@@ -60,8 +60,12 @@ MB.forms = {
 
     linkTypeOptions: function (root, backward) {
         function getText(data) {
-            return stripAttributes(data, l_relationships(
-                backward ? data.reverse_link_phrase : data.link_phrase));
+            return stripAttributes(
+                data, 
+                l_relationships(
+                  backward ? data.reverse_link_phrase : data.link_phrase
```

Comment:
We generally indent 2 spaces but this particular file is 4, so. :) I also suggested adding in the trailing comma:
```suggestion
                    backward ? data.reverse_link_phrase : data.link_phrase,
```

Reply:
Okay this one is my bad my setting is 2 spaces indent so i didn't notice. But when i keep the trailing commas in object literals the selenium tests do not pass and i tested with chrome dev tools that if i add trailing comma it says missing '('.

Reply:
We use trailing commas in object literals in tons of places, so that can't be right. :) I checked https://circleci.com/gh/metabrainz/musicbrainz-server/7375 and don't see any error about a missing "(" (unless I'm missing something too). Chrome definitely supports it, too, so that must've been an unrelated syntax issue.

Reply:
Yeah i'm so confused i know that trailing commas is supported in object literals but removing it makes the tests pass somehow i think i made a mistake somewhere else so let me doublecheck it. :DD